### ISSUE TITLE
Paragraph classifier

### ISSUE THREAD

**Issue by hhaoyan** [lbnlp/lbnlp#3]
It would be nice to add the paragraph classifier (https://github.com/CederGroupHub/MatBERT-synthesis-classifier) to this package.

**Comment by ardunn** [comment]
@hhaoyan yes that would be great! Is there a way you can condense it into something that would fit in `models`?

The steps to doing so are:

1. Write your model's metadata in [this file](https://github.com/lbnlp/lbnlp/blob/master/lbnlp/models/modelpkg_metadata.json), including a link to weights uploaded to a publicly accessible resource (e.g., figshare)
2. Write some classes for loading the pertinent classes of the package, once downloaded and installed, and putting them in this [directory: ](https://github.com/lbnlp/lbnlp/tree/master/lbnlp/models/load). See the files in there for some examples, but it can be pretty flexible
3. Update the docs with some simple examples of how to use it (so us other text mining groups can use it easily!)

Tag me if you need any help!
